app/src/examples/PendulumExample.tsx

```diff
@@ -0,0 +1,233 @@
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withSpring,
+} from 'react-native-reanimated';
+import { StyleSheet, TextInput, Text } from 'react-native';
+import { View, Button } from 'react-native';
+import React, { Dispatch, SetStateAction, useState } from 'react';
+import {
+  Gesture,
+  GestureDetector,
+  GestureHandlerRootView,
+  GestureStateManager,
+  GestureTouchEvent,
+} from 'react-native-gesture-handler';
+
+type FieldDefinition = [
+  string,
+  string,
+  Dispatch<SetStateAction<string>>,
+  Dispatch<SetStateAction<number>>
+];
+
+function InputField({
+  fieldName,
+  textValue,
+  setTextValue,
+  setValue,
+}: {
+  fieldName: string;
+  textValue: string;
+  setTextValue: Dispatch<SetStateAction<string>>;
+  setValue: Dispatch<SetStateAction<number>>;
+}) {
+  return (
+    <View style={styles.row} key={fieldName}>
+      <Text
+        style={[
+          styles.inputDescription,
+          {
+            textDecorationLine: fieldName === 'Duration' ? 'underline' : 'none',
+            fontWeight: fieldName === 'Duration' ? 'bold' : undefined,
+          },
+        ]}>
+        {fieldName}
+      </Text>
+      <TextInput
+        key={fieldName}
+        value={textValue}
+        onChangeText={(s) => {
+          const parsedInput = Number.parseFloat(s);
+          if (parsedInput) {
+            setValue(parsedInput);
+          }
+          setTextValue(s);
+        }}
+        autoCapitalize="none"
+        inputMode="numeric"
+        style={[styles.input]}
+      />
+    </View>
+  );
+}
+
+export default function SpringExample(): React.ReactElement {
+  const pendulumSwing = useSharedValue(0);
+  const offset = useSharedValue({ x: 0, y: 0 });
+  const [useConfigWithDuration, setUseConfigWithDuration] = useState(true);
+
+  const [stiffnessText, setStiffnessText] = useState('100');
+  const [stiffness, setStiffness] = useState(100);
+
+  const [durationText, setDurationText] = useState('5000');
+  const [duration, setDuration] = useState(5000);
+
+  const [dampingRatioText, setDampingRatioText] = useState('0.5');
+  const [dampingRatio, setDampingRatio] = useState(0.5);
+
+  const [massText, setMassText] = useState('1');
+  const [mass, setMass] = useState(1);
+
+  const [dampingText, setDampingText] = useState('10');
+  const [damping, setDamping] = useState(1);
+
+  const config = {
+    stiffness: stiffness,
+    ...(useConfigWithDuration
+      ? {
+          duration: duration,
+          dampingRatio: dampingRatio,
+        }
+      : { mass: mass, damping: damping }),
+  };
+
+  const style = useAnimatedStyle(() => {
+    return {
+      transform: [
+        { translateY: -100 },
+        { rotateZ: `${pendulumSwing.value}deg` },
+        { translateY: 100 },
+      ],
+    };
+  });
+
+  const gesture = Gesture.Pan()
+    .manualActivation(true)
+    .onChange((e) => {
+      'worklet';
```

Comment:
There's no need to mark these callbacks as worklets since they will be auto-workletized.